packages/bot-web-ui/src/components/dashboard/dashboard-component/dashboard-component.tsx

```diff
@@ -28,11 +28,35 @@ const DashboardComponent = ({
                 <div className='tab__dashboard__content'>
                     <div className='tab__dashboard__centered'>
                         {!has_dashboard_strategies && (
-                            <div className='tab__dashboard__header'>
-                                <Text color='prominent' line_height='xxl' size='m' weight='bold'>
-                                    {localize('Load or build your bot')}
-                                </Text>
-                            </div>
+                            <>
+                                <div className='tab__dashboard__header'>
+                                    <Text color='prominent' line_height='xxl' size='m' weight='bold'>
+                                        {localize('Load or build your bot')}
+                                    </Text>
+                                </div>
+                                <div className='tab__dashboard__home__retrigger'>
+                                    <button
+                                        onClick={() => {
+                                            setActiveTab(4);
+                                        }}
+                                    >
```

Comment:
```suggestion
                                    <button onClick={() => setActiveTab(4)}>
```
🙂 nit pick: please check if this will fit in one line after prettifying with option+shift+f (vscode)

Reply:
Done ✅